eng/config/PublishData.json

```diff
@@ -47,6 +47,16 @@
       "Microsoft.VisualStudio.IntegrationTest.Utilities": "vs-impl",
       "Microsoft.VisualStudio.LanguageServices.DevKit": "npm",
       "Microsoft.CodeAnalysis.LanguageServer": "vs-impl",
```

Comment:
We are leaving this old entry, because the version of this file from the main branch is the one always used for publishing. So if any servicing branch did a signed build, it'd look for this entry or otherwise it'll fail.